src/aliceVision/image/imageAlgo.cpp

```diff
@@ -151,8 +151,7 @@ void colorconvert(oiio::ImageBuf& imgBuf, image::EImageColorSpace fromColorSpace
     {
         if(fromColorSpace == EImageColorSpace::SRGB)
             oiio::ImageBufAlgo::colorconvert(imgBuf, imgBuf,
-                                             EImageColorSpace_enumToOIIOString(EImageColorSpace::SRGB),
-                                             EImageColorSpace_enumToOIIOString(EImageColorSpace::LINEAR));
+                                             EImageColorSpace_enumToString(EImageColorSpace::SRGB), EImageColorSpace_enumToString(EImageColorSpace::LINEAR));
```

Comment:
Well, not break in this particular case, but we should consistently use EImageColorSpace_enumToOIIOString to pass colorspaces to OIIO, because other enum values may be different.

Reply:
Ouch, good catch!